utils/utils.go

```diff
@@ -61,6 +61,10 @@ func GetChildNumsFromHex(hexHash string) ([]uint32, error) {
 		if num > maxInt32 {
 			num = num - maxInt32
 		}
+		// todo: @siggi is this a safe change? re: https://github.com/BuxOrg/bux/issues/4
+		if num > maxInt32 {
```

Comment:
This is safe through assumption ... the num is parsed as a Uint32, so can never be bigger than 2 * Int32.